### DISCUSSION TITLE
update - mechanical

### DISCUSSION THREAD

**Discussion by gzweigle** [gzweigle/open-hybrid-piano#22] [Announcements]
Testing note-to-note consistency.
Needs improvement.
Working on better mechanical system.
Also, built and testing an optional CNY-70 calibration system (partially in repo).

**Comment by edenchan2023** [comment]
Hello sir,

I have been preparing for my graduation thesis and came across your project (https://github.com/gzweigle/nonlinear_control) --《REAL-TIME OPTIMAL CONTROL OF LARGE SCALE POWER SYSTEMS WITH
APPLICATION TO IMPROVING TRANSIENT STABILITY RESPONSE》，while searching for reference papers. However, I couldn't find the code you provided for the project. Sir, I kindly request if you could take some time to upload the project code. I would be extremely grateful!

Thank you so much!
